doc/examples/motion_planning_python_api/motion_planning_python_api_tutorial.rst

```diff
@@ -0,0 +1,298 @@
+                panda_arm.execute()
+
+
+Single Pipeline Planning - Pose Goal
+==================================================
+Another common way to specify a goal state is via a pose goal ros message. Here we demonstrate how to set a pose goal for the end effector of the robot: ::
+
+        # set plan start state to current state
+        panda_arm.set_start_state_to_current_state()
+
+        # set pose goal with PoseStamped message
+        pose_goal = PoseStamped()
+        pose_goal.header.frame_id = "panda_link0"
+        pose_goal.pose.orientation.w = 1.0
+        pose_goal.pose.position.x = 0.28
+        pose_goal.pose.position.y = -0.2
+        pose_goal.pose.position.z = 0.5
+        pose_goal = {"link_name": "panda_link8", "pose": pose_goal}
+        panda_arm.set_goal(pose_goal=pose_goal)
+
+        # plan to goal
+        logger.info("Planning trajectory")
+        plan_result = panda_arm.plan()
+
+        # execute the plan
+        if plan_result:
+                logger.info("Executing plan")
+                panda_arm.execute()
+
+Single Pipeline Planning - Custom Constraints
+==================================================
+Arguably the most flexible way to specify a goal which we wish to plan for is via custom constraints, here we demonstrate planning to a configuration that satisfies a set of joint constraints: ::
+
+        # set plan start state to current state
+        panda_arm.set_start_state_to_current_state()
+
+        # set plan start state to current state
+        panda_arm.set_start_state_to_current_state()
+
+        # set constraints message
+        joint_values = {
+                "panda_joint1": -1.0,
+                "panda_joint2": 0.7,
+                "panda_joint3": 0.7,
+                "panda_joint4": -1.5,
+                "panda_joint5": -0.7,
+                "panda_joint6": 2.0,
+                "panda_joint7": 0.0,
+        }
+        robot_state.joint_positions = joint_values
+        joint_constraint = construct_joint_constraint(
+                robot_state=robot_state,
+                joint_model_group=moveit.get_robot_model().get_joint_model_group("panda_arm"),
+        )
+        panda_arm.set_goal(motion_plan_constraints=[joint_constraint])
+
+        # plan to goal
+        logger.info("Planning trajectory")
+        plan_result = panda_arm.plan()
+
+        # execute the plan
+        if plan_result:
+                logger.info("Executing plan")
+                panda_arm.execute()
+
+Multi Pipeline Planning
+===========================
+A recent addition to ``moveit_cpp`` and ``moveit_py`` is the ability to simultaneously execute multiple planning pipelines and select the resulting motion plan amongst all generated motion plans that best satisfies your task requirements. In previous sections, we defined a set of planning pipelines, here we will see how we can leverage these settings to simultaneously performing planning with multiple pipelines : ::
```

Comment:
I think using "parallel" instead of "simultaneous" is more precise and true to the new parallel planning feature. Something like this?

```suggestion
A recent addition to ``moveit_cpp`` and ``moveit_py`` is the ability to execute multiple planning pipelines in parallel and select the resulting motion plan amongst all generated motion plans that best satisfies your task requirements.
In previous sections, we defined a set of planning pipelines.
Here we will see how to plan in parallel with several of these pipelines : ::
```